roles/rp/tasks/main.yml

```diff
@@ -37,7 +37,7 @@
   sudo: true
   docker_container:
     name: rp
-    image: jrgm/123done
+    image: jrgm/123done{{ ':' + rp_docker_tag }}
```

Comment:
Yeah, 123done is not currently set up for circle-ci. It has a Dockerfile, which I built in my private dockerhub account.

Actually, maybe building with circle-ci isn't needed. There aren't any tests anyways and we just need a image that gets rebuilt on push, which mozilla dockerhub can do fine.